src/tools/fuzzing/fuzzing.cpp

```diff
@@ -3673,8 +3701,10 @@ Expression* TranslateToFuzzReader::makeCompoundRef(Type type) {
         builder.makeConst(int32_t(upTo(fuzzParams->MAX_ARRAY_SIZE)));
       return builder.makeArrayNew(type.getHeapType(), count, init);
     }
-    case HeapTypeKind::Cont:
-      WASM_UNREACHABLE("TODO: cont");
+    case HeapTypeKind::Cont: {
+      auto funcType = Type(heapType.getContinuation().type, NonNullable);
+      return builder.makeContNew(heapType, makeRefFuncConst(funcType));
```

Comment:
Why not allow this to be an arbitrary expression with the expected type?

Reply:
Done.